# commonly shared configurations among nodes
global_frame   : ar_global
odom_frame     : odom
base_frame     : base_footprint
nav_target_vm  : nav_target_vm
